source/includes/table-beta-compatibility.rst

```diff
@@ -50,11 +56,29 @@
      - |checkmark|
      - 
      - |checkmark|
+     - |checkmark|
 
    * - Destination Data Handling 
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
      - |checkmark|
-     - 
\ No newline at end of file
+     - 
+     - |checkmark|
+
+   * - Pre-6.0 Version Support
+     - |checkmark|
+     - |checkmark| [#pre-6.0-abc]_
+     - 
+     -
+     - |checkmark|
+     - |checkmark|
+     -
+
+.. [#pre-6.0-abc]
+
+   A->B->C migrations are compatible with pre-6.0 version support only if 
+   cluster A uses a server version older than 6.0. If cluster B uses a pre-6.0 
+   cluster, the second migration (B->C) can start only after the first migration 
+   (A->B) commits.
```

Comment:
```suggestion
   (A->B) committed.
```

Just to make it clear that this does not mean that the user can start their second migration after just issuing the /commit endpoint to the first migration

Reply:
gotcha! updating.